updating Release notes (#7572)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## June 2025

> - **New:** Added support for the bigquery_v1 adapter in the Semantic Layer and MetricFlow. This initial implementation supports authentication via native OAuth and Service Account JSON.

Reviewer: ```suggestion
- **New:** Added support for the bigquery_v1 adapter in the Semantic Layer and MetricFlow. This initial implementation supports authentication via native OAuth and Service Account JSON. Note: Workload Identity Federation is not yet supported.
```

>   🔒 Note: Workload Identity Federation (WIF) is not yet supported.

Reviewer: ```suggestion
```

- **New:**  Snowflake External OAuth is now supported for Semantic Layer queries.

> Snowflake connections that use External OAuth for user credentials can now emit queries for Insights through the Semantic Layer Gateway—enabling secure, identity-aware access via providers like Okta or Microsoft Entra ID.

Reviewer: ```suggestion
Snowflake connections that use External OAuth for user credentials can now emit queries for Insights through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.
```

## June 2025

> - **New:** Added support for the bigquery_v1 adapter in the Semantic Layer and MetricFlow. This initial implementation supports authentication via native OAuth and Service Account JSON. Note: Workload Identity Federation is not yet supported.

Author: ```suggestion
- **New:** Added support for the latest version of BigQuery credentials in Semantic Layer and MetricFlow.
```

- **New:**  Snowflake External OAuth is now supported for Semantic Layer queries.

> Snowflake connections that use External OAuth for user credentials can now emit queries for Insights, Cloud CLI, and Studio through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.

Author: ```suggestion
Snowflake connections that use External OAuth for user credentials can now emit queries for <Constant name="query_page" />, <Constant name="cloud_cli" />, and <Constant name="cloud_ide" /> through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.
```

> Snowflake connections that use External OAuth for user credentials can now emit queries for Insights, Cloud CLI,  and Studio through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.

Author: ```suggestion
Snowflake connections that use External OAuth for user credentials can now emit queries for Insights, Cloud CLI, and Studio through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.
```

> Snowflake connections that use External OAuth for user credentials can now emit queries for Insights through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.

Author: ```suggestion
Snowflake connections that use External OAuth for user credentials can now emit queries for Insights, Cloud CLI,  and Studio through the Semantic Layer Gateway. This enables secure, identity-aware access via providers like Okta or Microsoft Entra ID.
```
